python/oak/mapping_visu.py

```diff
@@ -173,21 +173,33 @@ def removeKeyFrame(self, keyFrameId):
 def parseArgs():
     import argparse
     p = argparse.ArgumentParser(__doc__)
-    p.add_argument("--dataFolder", help="Folder containing the recorded session for mapping")
-    p.add_argument("--recordingFolder", help="Folder for recording the input data for offline replay")
+    p.add_argument("--dataFolder", help="Instead of running live mapping session, replay session from this folder")
+    p.add_argument("--recordingFolder", help="Record live mapping session for replay")
     p.add_argument("--outputFolder", help="Folder where to save the captured point clouds")
     p.add_argument("--voxel", help="Voxel size (m) for downsampling point clouds")
     p.add_argument("--manual", help="Control Open3D camera manually", action="store_true")
     p.add_argument("--smooth", help="Apply some smoothing to 3rd person camera movement", action="store_true")
     p.add_argument("--color", help="Filter points without color", action="store_true")
     p.add_argument("--use_rgb", help="Use OAK-D RGB camera", action="store_true")
     p.add_argument('--ir_dot_brightness', help='OAK-D Pro (W) IR laser projector brightness (mA), 0 - 1200', type=float, default=0)
+    p.add_argument("--useRectification", help="--dataFolder option can also be used with some non-OAK-D recordings, but this parameter must be set if the videos inputs are not rectified.", action="store_true")
     return p.parse_args()
 
 if __name__ == '__main__':
     args = parseArgs()
     if args.outputFolder:
         os.makedirs(args.outputFolder)
+
+    configInternal = {
+        "computeStereoPointCloud": "true",
+        "pointCloudNormalsEnabled": "true",
+        "computeDenseStereoDepth": "true",
+    }
+    if args.dataFolder and args.useRectification:
+        configInternal["useRectification"] = "true"
+    else:
+        configInternal["alreadyRectified"] = "true"
+
```

Comment:
This unrelated change makes the `mapping_visu.py` example work more reliably with recordings from different devices.